packages/mui-joy/src/styles/CssVarsProvider.test.tsx

```diff
@@ -111,18 +111,21 @@ describe('[Joy] CssVarsProvider', () => {
           800: 'var(--joy-palette-neutral-800)',
           900: 'var(--joy-palette-neutral-900)',
           textColor: 'var(--joy-palette-neutral-textColor)',
+          textHoverColor: 'var(--joy-palette-neutral-textHoverColor)',
           textHoverBg: 'var(--joy-palette-neutral-textHoverBg)',
           textActiveBg: 'var(--joy-palette-neutral-textActiveBg)',
           textDisabledColor: 'var(--joy-palette-neutral-textDisabledColor)',
           outlinedColor: 'var(--joy-palette-neutral-outlinedColor)',
           outlinedBorder: 'var(--joy-palette-neutral-outlinedBorder)',
+          outlinedHoverColor: 'var(--joy-palette-neutral-outlinedHoverColor)',
           outlinedHoverBg: 'var(--joy-palette-neutral-outlinedHoverBg)',
           outlinedHoverBorder: 'var(--joy-palette-neutral-outlinedHoverBorder)',
           outlinedActiveBg: 'var(--joy-palette-neutral-outlinedActiveBg)',
           outlinedDisabledColor: 'var(--joy-palette-neutral-outlinedDisabledColor)',
           outlinedDisabledBorder: 'var(--joy-palette-neutral-outlinedDisabledBorder)',
           lightColor: 'var(--joy-palette-neutral-lightColor)',
           lightBg: 'var(--joy-palette-neutral-lightBg)',
+          lightHoverColor: 'var(--joy-palette-neutral-lightHoverColor)',
```

Comment:
Are we intentionally not having `containedHoverColor`?

Reply:
I think @danilo-leal will tweak the colors again.

Reply:
I will!